lib/engine/game/g_18_usa/game.rb

```diff
@@ -668,6 +668,13 @@ class Game < G1817::Game
         BRIDGE_CITY_HEXES = %w[I19 C10 C17 D14 E15 E17 F20 G17].freeze
         BRIDGE_TILE_HEXES = %w[B4 D18 E21 F18 H18].freeze
 
+        # Does not include guaranteed metropolis New York City
+        POTENTIAL_METROPOLIS_HEXES = %w[D20 E11 G3 H14 H22 I19].freeze
+
+        def potential_metropolitan_hexes
+          POTENTIAL_METROPOLIS_HEXES
```

Comment:
Recommend converting to Hex objects here and caching the result.

Reply:
Changed